core/src/main/java/apoc/export/csv/CsvFormat.java

```diff
@@ -281,20 +290,21 @@ private void writeRow(ExportConfig config, ExportFileManager writer, Set<String>
         }
     }
 
-    private List<String> generateHeader(Map<String, Class> propTypes, boolean useTypes, String... starters) {
-        List<String> result = new ArrayList<>();
-        if (useTypes) {
-            Collections.addAll(result, starters);
-        } else {
-            result.addAll(Stream.of(starters).map(s -> s.split(":")[0]).collect(Collectors.toList()));
-        }
+    private List<Map.Entry<String, String>> generateHeader(Map<String, Class> propTypes, boolean useTypes, String... starters) {
```

Comment:
Any reason why you have changed the type to `List<Map.Entry<String, String>`?

Reply:
Because this way I have a list of entries `<PropertyName, PropertyDataType>`,
which I think is easier to handle in later lines, 
i.e. the `writeNext()` on line 170 (where I use both `PropertyName` and `PropertyDataType` ) 
and in the `getNamesHeader(..)`, where I use `PropertyName` only, without needing to split the ":" to retrieve the names of the properties